docker/build/devdeps.Dockerfile

```diff
@@ -138,8 +113,8 @@ ENV HOME=/home SHELL=/bin/bash LANG=C.UTF-8 LC_ALL=C.UTF-8
 ENV SETUPTOOLS_SCM_PRETEND_VERSION=0.0.0
 
 # Copy over the llvm build dependencies.
-COPY --from=llvmbuild /opt/llvm /opt/llvm
-ENV LLVM_INSTALL_PREFIX=/opt/llvm
+COPY --from=prereqs /usr/local/llvm /usr/local/llvm
```

Comment:
Should `/usr/local` be hardcoded here?

Reply:
1) I took care that the paths in different files are independent, i.e. the path set in one file should be irrelevant for another file (that's what environment variables are for)
2) I don't think COPY supports anything that is not hardcoded: https://github.com/moby/moby/issues/34482

Reply:
I take 1 to mean that you are trying to use environment variables to convey file paths between different scripts and layers of scripts.
On the other hand, this case violates that approach.
On the third hand, 2 suggests the actual reason the tactic in 1 fails to apply here. docker's COPY command is incapable of properly handling environment variables.

Ok.

Reply:
Yes, exactly.